### ISSUE TITLE
Possible Duplicate: LIT4696MagicPr and LIT5679MagicPrayer

### ISSUE THREAD

**Issue by smaugustine** [BetaMasaheft/Documentation#1477]
The entry [LIT4696MagicPr](https://betamasaheft.eu/works/LIT4696MagicPr/main), which was created for BAVet 99, is for the prayer typically known as ልሳነ፡ ሰብእ፡ or መልክአ፡ ልሳን፡ (ChRep 314).

It is unclear whether [LIT5679MagicPrayer](https://betamasaheft.eu/works/LIT5679MagicPrayer) is a duplicate of this or if it is rather a different prayer with the same incipit, namely, ልሳነ፡ ያዕቆብ፡ (edited by Lifchitz). LIT5679MagicPrayer is only witnessed in two Russian manuscripts, and when I checked the catalogue there was not enough text given to determine which prayer it is. Is it possible to check whether these two manuscripts contain ልሳነ፡ ሰብእ፡ or ልሳነ፡ ያዕቆብ፡?

I suppose, in any case, that LIT4696MagicPr could be used for ልሳነ፡ ሰብእ፡ and LIT5679MagicPrayer for ልሳነ፡ ያዕቆብ፡ (and the manuscript records adjusted accordingly). Does that seem sensible? I know of at least one manuscript already on BetMas that definitely contains ልሳነ፡ ያዕቆብ፡ and could be added as a witness.

**Comment by eu-genia** [comment]
In any case, if LIT4696MagicPr is ChRep314 it is the doublet of LIT3034RepCh314 and should be deleted/merged

**Comment by eu-genia** [comment]
please feel free to 
- move/merge the content to LIT3034RepCh314 and add there 
`<relation name="betmas:formerlyAlsoListedAs" active="LIT3034RepCh314" passive="LIT4696MagicPr"></relation>`
- delete LIT4696MagicPr and 
- correct the BAVet99 description

as for LIT5679MagicPrayer - @DariaElagina should control the record 

@PietroLiuzzo while we are at it, can you check why the title under LIT5679 (`<title xml:lang="en" xml:id="t2">Magic prayer against the tongue of people</title>`) is printed as 
![image](https://user-images.githubusercontent.com/17987728/92472810-4e79ef80-f1da-11ea-8c7d-5742c74392f8.png)
?

**Comment by eu-genia** [comment]
the PR is good
those who used LIT3034RepCh314 in their records (@thea-m FSUor133 @DenisNosnitsin1970 IVef36, and the EMIP ones?) should maybe make sure that they really refer to Lessana sab', it is strange that noone noticed that ChRep314 is not a "Hymn to Jesus Christ"

**Comment by smaugustine** [comment]
I checked a couple of them, but I wouldn't mind going through more, I have been looking for copies of the text anyway.

**Comment by smaugustine** [comment]
As a note, it is included in copies of መልክአ፡ ጉባኤ፡ (separate from "magical" manuscripts), since the structure of its stanzas is the same as a መልክእ፡. Each stanza contains a supplication to Christ, so presumably that is why it was considered a "hymn to Jesus Christ."

The PR needs a reviewer with write access in order to merge.

**Comment by PietroLiuzzo** [comment]
> The PR needs a reviewer with write access in order to merge.
not sure what you mean and which PR.... you have merged your own PR... please remember to use Squash and Merge and to delete the branch you have merged.

> title under LIT5679
 title should be fine now  "Ṣalot baʾǝnta ḥǝmāma lǝssāna sabʾ zamad" https://betamasaheft.eu/works/LIT5679MagicPrayer/main

**Comment by PietroLiuzzo** [comment]
I did not read this entire conversation, but the deletion was certainly too quick. It would be in my opinion better and more sensible practice to update the entries with an ID which is a candidate for removal before the ID is deleted, and need not to wait for others to do that. Waiting for the merge would have been beneficial, it was my fault, that I approved after adding @eu-genia as reviewer, and prompted a merge action. I did that just because I saw it and saw no problem, so I thought I'd spare some steps. nevermind, but please update the entries.

**Comment by DariaElagina** [comment]
Concerning LIT5679, I will try to get pictures of  [https://betamasaheft.eu/manuscripts/InEtn210327/main](url), which is for free and not too complicated. However, it might take time due to the pandemic.

**Comment by eu-genia** [comment]
@PietroLiuzzo i did not merge originally the PR (i was assigned to review and merge) because I first wanted the issue to be checked by the others involved and for the records involved to be updated

**Comment by PietroLiuzzo** [comment]
As said, it was my mistake, not yours.

**Comment by thea-m** [comment]
FSUor133 contains LIT3034RepCh314.

**Comment by smaugustine** [comment]
Following up on the question of ልሳነ፡ ያዕቆብ፡ ([edited by Lifchitz based on BnF Éth 354](https://gallica.bnf.fr/ark:/12148/bpt6k26162n/f250.image), also found in [Éth 387](https://gallica.bnf.fr/ark:/12148/btv1b10029933c/f9.item) and [EMIP 185](https://betamasaheft.eu/manuscripts/EMIP00185/main#ms_i7)), is it necessary to create a new work for this? I have looked for one and not found anything.

**Comment by thea-m** [comment]
Here it is: LIT1771Lessan (you were probably searching with fidal? It's always good to try transliteration as well, in cases such as this where the concerned file has not yet been edited)

**Comment by smaugustine** [comment]
LIT1771Lessan is still for ልሳነ፡ ሰብእ፡ (aka መልክአ፡ ልሳን፡).

ልሳነ፡ ያዕቆብ፡ is a separate text (based on Genesis 30-1). It often has a similar incipit (ጸሎት፡ በእንተ፡ ልሳነ፡ ሰብእ፡ መቀኛ፡ &c.), but is totally different. EMIP 185 contains both -- I'm sure I've seen other MSS that contain both, but can't recall any specific examples.

Importantly, ልሳነ፡ ሰብእ፡ is found both in "magical" MSS and in መልክአ፡ ጉባኤ፡, whereas ልሳነ፡ ያዕቆብ፡ would only occur in the former.

**Comment by eu-genia** [comment]
@smaugustine  since the IDs follow the EAe index you can easily verify (I have the source file, and the Excel, on the basis of which the stubs were created) that LIT1770 is indeed ልሳነ፡ ሰብእ፡ and LIT1771 is ልሳነ፡ ያዕቆብ፡
90% records were inherited from the EAe, and of these 90% were never touched/edited so have no fidal and EAe-style transcription

**Comment by thea-m** [comment]
LIT1771Lessan has  `<title xml:id="t1">Lǝssanä Yāʿqob</title>` (and nothing else) in the master branch. Are you in another branch where it has been edited maybe? Or there might be confusion with LIT1770Lessan?

**Comment by smaugustine** [comment]
I see, I see. Is LIT1771 still usable? I see its file in the Works repo, but attempting to visit the url shows that it "is not an available item."

**Comment by PietroLiuzzo** [comment]
yes. the id is also composed of the characters after the numeric part

**Comment by smaugustine** [comment]
I always forget this 🤦🏻‍♂️! Thank you!
